gym/export_model.sh

```diff
@@ -0,0 +1,15 @@
+echo "Exporting core model"
```

Comment:
@BaniaFonseca @tjiagoM : Going back to the MLFlow applicability. This would be in an ideal scenario the right place to add it as a Model Registry. If we had one accessible from somewhere.

So the API side would just need to retrieve the model from a registry with interrelated metadata and versioning information.

My initial biggest concern is that we needed hosting, I am currently trialling we can use the community edition self hosted version.

Reply:
Maybe you can this out in a new branch, because I did not get it ...